docs/llm/reference.md

```diff
@@ -96,6 +96,8 @@ Consider increasing the `cache_size` parameter in case the logs report the usage
 `enable_prefix_caching` can improve generation performance when the initial prompt content is repeated. That is the case with chat applications which resend the history of the conversations. Thanks to prefix caching, there is no need to reevaluate the same sequence of tokens. Thanks to that, first token will be generated much quicker and the overall
 utilization of resource will be lower. Old cache will be cleared automatically but it is recommended to increase cache_size to take bigger performance advantage.
 
+`dynamic_split_fuse` [algorithm](https://github.com/microsoft/DeepSpeed/tree/master/blogs/deepspeed-fastgen#b-dynamic-splitfuse-) is enabled by default to boost the throughput by splitting the tokens to even chunks. In some conditions like with very low concurrency or with very short prompts, it might be beneficial to disable this algorithm. When it is disabled, there should be set also the parameter `max_num_batched_tokens` to match the model max context length.
```

Comment:
Are we sure about this recommendation? 
@ilya-lavrenov has vLLM scheduling some advantage with low concurrency or short prompts?

Reply:
@mzegla @ilya-lavrenov based on benchmarks dynamic split fuse boost the throughput with medium and high concurrency by about 20% but also increase latency in a low concurrency by about 20%. In most cases the throughput is most important but in use case of a single client, disabling it might give a bit better results. Maybe some optimizations could be considered to optimize it so dynamic split fuse could be the only path...